build

```diff
@@ -109,3 +126,14 @@ for project in filter(
     module = module.lstrip("lib") if module.startswith("lib") else module
     dst_file = os.path.join(QUERY_MODULES_DIRECTORY, module)
     shutil.copy2(src_file, dst_file)
+
+#################################################
+#            Optional copy dist folder          #
+#################################################
+
+arguments = get_arguments()
+
+if arguments.path is not None:
```

Comment:
Is there a better way to check if parameter is existing in argparse?

Reply:
I will check this out.

Reply:
I found a slightly better way.

Reply:
Check if it is okay now.